### ISSUE TITLE
Selected text does not refresh when the language of options changed 

### ISSUE THREAD

**Issue by chencc1988** [siemens/ix#172]
## I'm submitting an i18n requirement

<pre><code>
[ ] Bug report
[x] Feature request
[ ] Documentation issue or request
</code></pre>

## Current behavior

I tried to use ``@ngx-translate/core`` to implement the i18n feature, and the translate service provide an observable to monitor the language changing events. 
The options' label follow the change of language as expected, however the selected item label still shows the old language unless you click the item once more.

## Expected behavior

The label text of selected item keep the same to options

## Minimal reproduction of the problem with instructions

```
import { Component, OnInit } from '@angular/core';
import { BehaviorSubject } from 'rxjs';

@Component({
  selector: 'app-testpage',
  template: `<ix-select [selectedIndices]="selectedIndices">
      <ng-container *ngFor="let item of options">
        <ix-select-item
          [label]="item.label$ | async"
          [value]="item.value"
        ></ix-select-item>
      </ng-container>
    </ix-select>
    <ix-button class="m-1" variant="primary" (click)="useLanguage1()"
      >English</ix-button
    >
    <ix-button class="m-1" variant="primary" (click)="useLanguage2()"
      >German</ix-button
    >`,
})
export class TestpageComponent implements OnInit {
  selectedIndices = '1';
  subject$ = new BehaviorSubject<string>('Item 1');
  options = [{ value: '1', label$: this.subject$.asObservable() }];
  constructor() {}

  ngOnInit(): void {}

  useLanguage1() {
    this.subject$.next('Item 1');
  }

  useLanguage2() {
    this.subject$.next('Punkt 1');
  }
}

```

## What is the motivation / use case for changing the behavior?

For i18n reason

**Comment by chencc1988** [comment]
It seems that the PR not amend this issue. Still the same in Angular.
![Animation](https://user-images.githubusercontent.com/10339536/207545942-3f8ffd02-e0bb-428e-b062-a2557e1311a6.gif)

**Comment by danielleroux** [comment]
@chencc1988 please look here: https://stackblitz.com/edit/angular-qo5vdv?file=package.json,src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html

**Comment by chencc1988** [comment]
@danielleroux 
I did a minor change that move the html template part into a ix-menu-settings-item, then it stop work
https://stackblitz.com/edit/angular-qo5vdv-7qn5in?file=src/app/app.component.html

**Comment by danielleroux** [comment]
@chencc1988 Should be fixed with #300. Will be in version 1.3.0.
